Lib/ast.py

```diff
@@ -508,27 +508,6 @@ def generic_visit(self, node):
             elif isinstance(value, AST):
                 self.visit(value)
 
-    def visit_Constant(self, node):
```

Comment:
U*h I think you accidentally removed this. Above said to use this. 😄

Reply:
I didn't accidentally remove it. It will still work if you define `visit_Constant` on a custom subclass of `ast.NodeVisitor`. The reason why it is currently defined on `ast.NodeVisitor` itself — uniquely among the `visit_*` methods — is to ensure that the deprecated methods `visit_Num`, `visit_Str`, etc. continue to work if you define them on a subclass of `ast.NodeVisitor`.

Reply:
I tried to clarify this a bit in https://github.com/python/cpython/pull/119563/commits/40b596553653941ec4a349deb503e075fe557439 -- hopefully that helps!